### ISSUE TITLE
Error in test_aws_security_group_inline_rule_open and test_aws_security_group_rule_open

### ISSUE THREAD

**Issue by brjlpz** [accurics/terrascan#138]
* terrascan version: 0.1.0
* Python version: Python 3.5.3
* Operating System: Debian 9

### Description

I get errors with the security group tests.

sample tf:
```
resource "aws_security_group_rule" "integration_allow_ssh_from_bastion" {
  security_group_id        = "${aws_security_group.integration.id}"
  source_security_group_id = "${aws_security_group.bastion.id}"
  from_port                = 22
  to_port                  = 22
  protocol                 = "tcp"
  protocol                 = -1
  type                     = "ingress"
}

```

### What I Did

```
terrascan --location . --tests all
...
Running security_group Tests
...EE
======================================================================
ERROR: test_aws_security_group_inline_rule_open (terrascan.checks.security_group.TestSecurityGroups)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/blopez/.local/lib/python3.5/site-packages/terrascan/checks/security_group.py", line 52, in test_aws_security_group_inline_rule_open
    'cidr_blocks').list_should_not_contain('0.0.0.0/0')
  File "/home/blopez/.local/lib/python3.5/site-packages/terraform_validate/terraform_validate.py", line 80, in property
    if property_name in property.property_value.keys():
AttributeError: 'list' object has no attribute 'keys'

======================================================================
ERROR: test_aws_security_group_rule_open (terrascan.checks.security_group.TestSecurityGroups)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/blopez/.local/lib/python3.5/site-packages/terrascan/checks/security_group.py", line 44, in test_aws_security_group_rule_open
    'cidr_blocks').list_should_not_contain('0.0.0.0/0')
  File "/home/blopez/.local/lib/python3.5/site-packages/terraform_validate/terraform_validate.py", line 167, in list_should_not_contain
    actual_property_value = self.validator.substitute_variable_values_in_string(property.property_value)
  File "/home/blopez/.local/lib/python3.5/site-packages/terraform_validate/terraform_validate.py", line 504, in substitute_variable_values_in_string
    variable_default_value = self.get_terraform_variable_value(a.variable)
  File "/home/blopez/.local/lib/python3.5/site-packages/terraform_validate/terraform_validate.py", line 493, in get_terraform_variable_value
    raise TerraformVariableException("There is no Terraform variable '{0}'".format(variable))
terraform_validate.terraform_validate.TerraformVariableException: There is no Terraform variable ''

----------------------------------------------------------------------
```

**Comment by alexppg** [comment]
Also, when executing all the tests, the error code is 0, even when failing. When executing only the security_group test and fails, works as expected and returns 1.

**Comment by cesar-rodriguez** [comment]
This should be fixed as of Terrascan v1.0.0
